deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Product.wxs

```diff
@@ -6,7 +6,7 @@
   <Product Id="*"
            Name="$(var.ProductName)"
            Language="1033"
-           Version="$(var.InstallerVersion)"
+           Version="$(InstallerVersion)"
```

Comment:
This will result in a syntax error from the compiler.

Reply:
Yep, I noticed. It's fixed.